### ISSUE TITLE
bug: [Error] - :app:processDebugGoogleServices

### ISSUE THREAD

**Issue by renat473** [chatwoot/chatwoot-mobile-app#646]
hello everyone, I'm having problems trying to build and returning the following error message :app:processDebugGoogleServices I thought it could be the google-service.json file but it's in the correct directory oriented by the application.
Version Android
![image](https://user-images.githubusercontent.com/1757167/222930113-d2d82a18-0e38-4a4d-8228-243989acd8e7.png)

**Comment by renat473** [comment]
help @muhsin-k

**Comment by muhsin-k** [comment]
Can you check `google-services.json` file was added or not?
![CleanShot 2023-03-14 at 10 57 26](https://user-images.githubusercontent.com/12408980/224904796-4620534e-0f29-4302-b376-c98a0f036d77.png)

**Comment by renat473** [comment]
Yes, one of the main items I checked was the google-services.json file

![image](https://user-images.githubusercontent.com/1757167/225007467-e710591a-a4ed-47f2-b1c6-ed3f1a01b37d.png)

**Comment by renat473** [comment]
hello @muhsin-k 

Do you have any other suggestions I can check out?

**Comment by muhsin-k** [comment]
@renat473 It Seems like some build issues related to RN. Can you check if other react native projects are working or not? Try to scaffold a new RN project.

**Comment by renat473** [comment]
and the first project, I never got to do some kind of build.

Is there any minimum application configuration to be used @muhsin-k ?

**Comment by renat473** [comment]
OS: Windows 10 10.0.19044
    CPU: (4) x64 11th Gen Intel(R) Core(TM) i3-1115G4 @ 3.00GHz
    Memory: 1.81 GB / 7.73 GB
  Binaries:
    Node: 18.14.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 9.3.1 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK: Not Found
  IDEs:
    Android Studio: AI-213.7172.25.2113.9123335
    Visual Studio: Not Found
  Languages:
    Java: 11.0.16.1
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.1.0 => 18.1.0
    react-native: 0.70.6 => 0.70.6
    react-native-windows: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

**Comment by renat473** [comment]
any tips on what i can try @muhsin-k ?
